docs/jax.rst

```diff
@@ -99,3 +99,12 @@ Parallelization (:code:`pmap`)
     device_count
     local_device_count
     process_count
+
+Callbacks
+----------------------------
```

Comment:
We need the underline to match the length of the text (sphinx will error otherwise)

Reply:
Done.